src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -69,6 +69,7 @@
       font-size: em(22);
       outline: 0;
       line-height: 1.3;
+      min-height: em(17);
```

Comment:
Yes I verified on both the browsers and then set the height. It matches the edit mode of existing work items.